doto-backend/src/swagger.json

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
Would probably good to add some comments in this file describing it.

Reply:
Had a look online I do not think that you can comment  JSON Files.  But this is self documenting in the sense that once you open  /api-docs it will serve the contents of this JSON file in the form of UI.

Reply:
Right that makes sense. Disregard the commenting then.